config/filter.d/postfix.conf

```diff
@@ -24,7 +24,7 @@ mdre-normal=^%(_pref)s from [^[]*\[<HOST>\]%(_port)s: [45][50][04] [45]\.\d\.\d+
             ^from [^[]*\[<HOST>\]%(_port)s:?
 
 mdpr-auth = warning:
-mdre-auth = ^[^[]*\[<HOST>\]%(_port)s: SASL ((?i)LOGIN|PLAIN|(?:CRAM|DIGEST)-MD5) authentication failed:(?! Connection lost to authentication server| Invalid authentication mechanism)
+mdre-auth = ^[^[]*\[<HOST>\]%(_port)s: SASL ((?i)LOGIN|PLAIN|(?:CRAM|DIGEST)-MD5) authentication failed:(?! Connection lost to authentication server| ?!Invalid authentication mechanism)
```

Comment:
Adding `?!` here caused vice versa situation (it'd make space optional and `!` mandatory, don't confuse it with `(?!..,)` so that chars after parenthesis) - `(?! ...)` is a negative lookahead, that means that it would match `authentication failed`, **but not if it is followed** by `connection lost` or `invalid mechanism`.
So to say it'd start to work like `mdre-auth2` (used in aggressive mode only).
This was intentional and the reason for that - in normal mode the filter should consider only real authentications failures (for example if some application trying different methods or authentication mechanisms).
There is also a todo:
- check/remove "Invalid authentication mechanism" from ignore list, if #1243 will get finished (see #1297).

Reply:
I'll try aggressive mode, thanks